assets/timber.scss.liquid

```diff
@@ -1685,32 +1740,26 @@ label.error {
 .site-header {
   padding: ($gutter / 2) 0;
 
-  @include at-query ($min, $large) {
-    padding: $gutter 0;
-  }
+  .grid {
```

Comment:
This just removes the media query around the styles so that elements always use the table layout. Agreed though, will use variation class.